dashboard/extract_s3.py

```diff
@@ -0,0 +1,55 @@
+"""
+Extracts the archived csv files from bucket 
+into the archived_data folder.
+"""
+import os
+from os import environ
```

Comment:
surely importing the entire of os would mean you are already importing environ and path? May have to call as os.environ, os.path

Reply:
Here's the updated imports: from os import environ, path, makedirs
from dotenv import load_dotenv
from boto3 import client
from botocore.client import BaseClient